go-controller/pkg/util/multi_network.go

```diff
@@ -279,7 +279,7 @@ func newLocalnetNetConfInfo(netconf *ovncnitypes.NetConf) (NetInfo, error) {
 
 	ni := &secondaryNetInfo{
 		netName:        netconf.Name,
-		topology:       types.Layer2Topology,
```

Comment:
what about L275? I know that `parseSubnets` treats the cases for `Layer2Topology` and `LocalnetTopology` the same way so no functional changes but still feels like we should pass `types.LocalnetTopology` there as well since this is the `newLocalnetNetConfInfo` ?